docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,101 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
 - You can choose dedicated IOPs EBS drives on AWS and specify the number of dedicated IOPS you need.
 
 YugabyteDB Anywhere also allows creating these machines out of band and importing them as an on-premises install.
+
+## Node agent
+
+### What is a node agent?
+
+Node agent is an RPC service running on a YugabyteDB node, and is used to manage communication between YugabyteDB Anywhere and the nodes in universes. It includes the following functions:
+
+- Invoke shell commands directly on the remote host, similar to running commands over SSH. Like SSH, the agent also does shell-login such that any previous command modifying the environment in the resource files (for example, `~/.bashrc`) gets reflected in the subsequent command.
+- Invoke procedures or methods on the node agent.
+- Additionally, for on premises (manually provisioned nodes) deployments, node agent also functions as a utility to run preflight checks, and add node instances.
+
+### How is node agent installed on a YugabyteDB node?
+
+Installation depends on how the nodes are provisioned.
+
+For cloud (AWS, GCP, and Azure) and non-manual on-premises (sudo access provided) providers, agents are automatically installed on each universe node during provisioning using SSH. A sudo user with SSH access (for example, ec2-user for AWS) is required. After the node agent is installed, all the legacy SSH calls are replaced with node agent calls. Just like an SSH daemon, node agent is run as a root user to perform provisioning of the nodes. After node provisioning, you can revoke the SSH access, but it's recommended to enable the access for debugging.
+
+For manually provisioned on-premises providers, node agent is installed on each node either using a script or manually as part of the manual provisioning process. For more information, refer to [Manually provision on-premises nodes](../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual/).
+
+### Why does YBA prompt for SSH details if the node agent is installed manually and can replace SSH?
+
+When creating an on-premises provider, you are prompted to provide SSH credentials. After you have provisioned and added the instances to the provider (including installing the node agent), YugabyteDB Anywhere no longer requires SSH or sudo access to nodes.
+
+If you are manually provisioning nodes, these credentials aren't needed and you can provide a fictitious non-working SSH key.
+
+### What is registration and unregistration of node agent?
+
+Node agent is a secure service that authenticates every remote call from YBA. This requires the following:
+
+- Making YBA aware of the node agent service.
+- Establishing trust between the node agent and YBA.
+
+This process of establishing a secure connection is called registration. Note that registration has nothing to do with a provider.
```

Comment:
Note that, registration is making YBA and node aware of each other and trust each other. No provider level details are needed for registration.